code/DebugBarSilverStripeCollector.php

```diff
@@ -109,6 +109,9 @@ public static function getSessionData()
             if ($k === 'PHPDEBUGBAR_STACK_DATA') {
                 continue;
             }
+            if (is_array($v)) {
+                $v = json_encode($v, JSON_PRETTY_PRINT);
```

Comment:
PRETTY_PRINT is required here to make sure the output is properly formatted and ellipsed in the actual view.